### PR TITLE
Cellprofiler

### PR DESCRIPTION
**Pull request by ThomasWollmann**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/cellprofiler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - scipy
+    - sphinx
+    - tifffile
+    - wxpython=3.0.2.0
+    - cellh5
+    - centrosome
+    - inflect
+    - prokaryote==2.4.0
+    - javabridge==1.0.15
+    - python-bioformats==1.4.0
```
Comment (bgruening) [inline recipes/cellprofiler/meta.yaml]: Oh gosh, there is a difference between `bioformats` and `python-bioformats`?

File: recipes/cellprofiler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    - scikit-learn
+    - scipy
+    - sphinx
+    - tifffile
+    - wxpython=3.0.2.0
+    - cellh5
+    - centrosome
+    - inflect
+    - prokaryote==2.4.0
+    - javabridge==1.0.15
```
Comment (bgruening) [inline recipes/cellprofiler/meta.yaml]: This is only available in version 1.0.14 afaik and this is also more then a year old.

Comment (ThomasWollmann) [reply recipes/cellprofiler/meta.yaml]: @bgruening Do you think we can get this package to work somehow or is it too cumbersome?

**Comment by ThomasWollmann** [comment]
ping @jkh1

File: recipes/cellprofiler/meta.yaml (hunk 2 above)
Comment (bgruening) [reply recipes/cellprofiler/meta.yaml]: You mean javabridge?

Comment (ThomasWollmann) [reply recipes/cellprofiler/meta.yaml]: yes, since we need it for python-bioformats

**Comment by bgruening** [comment]
You need to relax the pinning I think.

**Comment by ThomasWollmann** [comment]
@bgruening For this package we still need https://github.com/CellH5/cellh5 https://github.com/CellProfiler/centrosome puhh could spend all night long packaging imaging software since nothing is available.

**Comment by ThomasWollmann** [comment]
@bgruening https://github.com/bioconda/bioconda-recipes/pull/16445

**Comment by ThomasWollmann** [comment]
@bgruening https://github.com/bioconda/bioconda-recipes/pull/16446

**Comment by ThomasWollmann** [comment]
Old PR

**Pull request closed without merging**